public/components/ctas/8.html

```diff
@@ -0,0 +1,26 @@
+<aside class="relative overflow-hidden text-gray-300 bg-gray-900 lg:flex justify-center">
+    <div class="w-full p-12 text-center lg:w-2/3 sm:p-16 lg:p-24">
+      <div class="max-w-xl mx-auto lg:ml-0">
+
+        <div class="p-2 border inline-block">community</div>
```

Comment:
Not a fan of having content without any inner markup within a `<div>`.

Replace with a `<span>` or a `<strong>`, your choice.